common/mariadb/values.yaml

```diff
@@ -25,7 +25,7 @@ db_performance_schema: # Enabling performance schema only (without enabling inst
   enabled: true
   
 db_performance_schema_instrument: # Enabling performance schema instruments. 
-                                  # In order to enable it you need to enable the enable_ps.
+                                  # Please enable the db_performance_schema in order to enable the db_performance_schema_instrument.
   enabled: false
   # default : false
```

Comment:
Ah, I am reading that as there is a key named "default", and it its value is set to "false", and it has be commented out. Possibly because it the same as not setting it